providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/model/FlagParser.java

```diff
@@ -42,15 +40,18 @@ public class FlagParser {
     private static JsonSchema SCHEMA_VALIDATOR;
 
     static {
-        try {
-            final URL url = FlagParser.class.getClassLoader().getResource(SCHEMA_RESOURCE);
```

Comment:
This was an issue identified while working with a local build library. Resource loader urls which are relative to jar contains exclamation mark, preventing file loading. This PR contains the fix.